docs/colors/helpers.js

```diff
@@ -0,0 +1,62 @@
+function hexToRGBArray(htmlHexColor) {
+  let hexColor = htmlHexColor.replace('#', '');
+
+  if (hexColor.length === 3) {
+    const [r, g, b] = hexColor;
+    hexColor = `${r}${r}${g}${g}${b}${b}`;
+  } else if (hexColor.length !== 6) {
+    throw new Error(`Invalid hex color: ${hexColor}`);
+  }
+
+  const rgb = [];
+  for (let i = 0; i <= 2; i += 1) {
+    rgb[i] = parseInt(hexColor.substr(i * 2, 2), 16);
+  }
+
+  return rgb;
+}
+
+function getLuminanace(r, g, b) {
+  const a = [r, g, b].map(value => {
+    const divided = value / 255;
+    return divided <= 0.03928
+      ? divided / 12.92
+      : ((divided + 0.055) / 1.055) ** 2.4;
+  });
+  return a[0] * 0.2126 + a[1] * 0.7152 + a[2] * 0.0722;
```

Comment:
Ok, so I think all this crazy numbers should go to descriptive consts.

Reply:
omg, it will require some research xD

Reply:
😂 you can also add a comment "based on ..."

Reply:
I used one method from `polished` and it is not needed anymore ;D